back/README.md

```diff
@@ -23,3 +23,23 @@ npm run start:debug
 ```
 
```

Comment:
It would great to add a short description about T-Shirt planning poker

Reply:
Yes, we should talk about that.